content/en/render-hooks/blockquotes.md

```diff
@@ -54,7 +54,7 @@ block = true
 (`string`) The position of the blockquote within the page content.
 
 ###### Text
-(`string`) The blockquote text, excluding the alert designator if present. See the [alerts](#alerts) section below.
+(`hstring.RenderedHTML`) The blockquote text, excluding the alert designator if present. See the [alerts](#alerts) section below.
```

Comment:
Note that the motivation behind this type is to have a way to detect circular render loops, which was a very common and very hard to debug user error.

I'm not totally sure the users need to bother with these technical details:

* In all practical situations it behaves like a `string`
* It renders as `template.HTML` which I guess is the most useful information.